js/src/jit/moz.build

```diff
@@ -28,6 +28,7 @@ UNIFIED_SOURCES += [
     "BitSet.cpp",
     "BytecodeAnalysis.cpp",
     "CacheIR.cpp",
+    "CacheIRAOT.cpp",
```

Comment:
When trying to build the project, it fails, stating this file does not exist - perhaps it was not committed or it's replaced with `CacheIRAOTGenerated.h`?